platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/style/expressions/ExpressionTest.java

```diff
@@ -1403,4 +1404,66 @@ public void testIsSupportedScriptExpressions() {
     Object[] actual = isSupportedScript(get("property_name")).toArray();
     assertTrue("expression should match", Arrays.deepEquals(expected, actual));
   }
+
+  @Test
+  public void testFormatSingleArgument() {
+    Object[] expected = new Object[] {"format", "test",
+      new HashMap<String, Object>() {
+        {
+          put("font-scale", 1.5f);
+          put("text-font", "awesome");
+        }
+      }
+    };
+    Object[] actual = format(literal("test"), literal(1.5), literal("awesome")).toArray();
+    assertTrue("expression should match", Arrays.deepEquals(expected, actual));
+  }
+
+  @Test
+  public void testFormatMultipleArgument() {
+    Object[] expected = new Object[] {
+      "format",
+      "test",
+      new HashMap<String, Object>() {
+        {
+          put("text-font", "awesome");
+        }
+      },
+      "test2",
+      new HashMap<String, Object>() {
+        {
+          put("font-scale", 1.5f);
+        }
+      },
+      "test3",
+      new HashMap<String, Object>() {
+        {
+        }
+      },
+      "test4",
+      new HashMap<String, Object>() {
+        {
+          put("font-scale", 1.5f);
+          put("text-font", "awesome");
+        }
+      }
+    };
+    Object[] actual = format(
+      literal("test"), null, literal("awesome"),
```

Comment:
`"awesome"` should be `new String[]{"awesome"}`
same applies for other occurrences of font-stack in the unit tests

Reply:
great catch!